package.json

```diff
@@ -40,6 +40,7 @@
     "react-native-hyperlink": "0.0.10",
     "react-native-i18n": "1.0.0",
     "react-native-image-picker": "0.26.3",
+    "react-native-keyboard-aware-scroll-view": "^0.3.0",
     "react-native-keyboard-spacer": "0.3.1",
```

Comment:
Could we remove this old an unused library?

Reply:
👍